lib/process-services-cloud/src/lib/form/services/form-cloud.service.ts

```diff
@@ -211,9 +211,10 @@ export class FormCloudService extends BaseCloudService implements FormCloudServi
      * @param json JSON data to create the form
      * @param data Values for the form's fields
      * @param readOnly Toggles whether the form should be read-only
+     * @param formService Implements Process Services form methods
      * @returns Form created from the JSON specification
      */
-    parseForm(json: any, data?: TaskVariableCloud[], readOnly: boolean = false): FormModel {
+    parseForm(json: any, data?: TaskVariableCloud[], readOnly: boolean = false, formService?: FormService): FormModel {
```

Comment:
This can be a private field here instead of the public (potential breaking change).
```
private formService = inject(FormService)
```